README.md

```diff
@@ -1,655 +1,288 @@
 
-# Uses GD still so need to do this first one separately so it can log in
-scipiper::scmake("1_fetch/out/dv_data.rds.ind", "1_fetch.yml")
+Note that before you can just build this as the code suggests, you will have to have the appropriate permissions to our intermediate data products. Mainly, this is access to the Google Drive folder which we grant to internal users and close collaborators.
 
-# Then all the rest
-scipiper::scmake("1_fetch/out/dv_data_fixed_gh.rds.ind", "1_fetch.yml")
-scipiper::scmake('1_fetch/out/sites_stage.rds.ind', remake_file = '1_fetch.yml')
-scipiper::scmake("2_process/out/dv_stats.rds.ind", "2_process.yml")
-scipiper::scmake("2_process/out/dv_stats_fixed_gh.rds.ind", "2_process.yml")
-scipiper::scmake("2_process/out/dv_stat_styles.rds.ind", "2_process.yml")
+## How to build this animation
 
-#####################
-## Create a callouts file (it can be empty if there are none)
+The process to create this animation is almost entirely automated using the (now dormant) custom dependency management R package, `scipiper`. There is a bit of manual work required to create and then prepare the animation's event/text callouts. Otherwise, it is mostly running chunks of code. The entire process is outlined below (*steps that aren't code, but are human checks are italicized in parentheses*).
 
-file.create("callouts_cfg.yml")
+1. Change the animation dates and download/process the new data (*notify Web Comms and GWSIP team that this process has begun*)
+1. Create a new blank animation to use for event callout inspiration
+1. Gather event callouts and add to the animation (*work with GWSIP collaborators & involve IIDD reviewers as necessary*)
+1. Tweak event callout timing and appearance
+1. Generate final video animation (*get it approved by IIDD reviewers during this step, after approval ask Web Comms to start the video description paragraph*)
+1. Run code for simple-to-produce outreach artifacts (*ask Web Comms to upload the VisID thumbnail to Drupal, then submit Drupal video upload form with link to that thumbnail*)
+1. Restructure code and build a new Instagram version (*share this and other outreach artifacts with Web Comms so the release package can be ready*)
 
-#####################
-## Build ALL frames and then make video
+*I plan to come back later and add instructions for how to generate the full water year animation*
 
-# Prior to building the frames:
-#   1. Make sure to delete the contents of `callouts_cfg.yml` before building or you will have old events on the new frames. 
-#   2. Make sure the title in `viz_config.yml` is correct.
+The outputs from following this full process should be the following files:
 
-# To make absolutely sure that your video will use the right font, you may need to run:
-sysfonts::font_add_google('Abel','abel')
+* `6_visualize/out/river_conditions_[month start]_[month end]_[year]_twitter.mp4`
+* `6_visualize/out/river_conditions_[month start]_[month end]_[year]_visid.mp4`
```

Comment:
what is the difference between visid and the social media versions? we should have visId on all of them, so maybe it is not needed to have a separate version for that, rather be a part of all versions. This comment is coming from an uninformed place and maybe is irrelevant

Reply:
Visid is the black bar across the bottom. We have been releasing on social media without that requirement, but including on the video on Drupal